scripts/hooks/baseline.py

```
@@ -0,0 +1,130 @@
+    path = _BASELINES_DIR / f"{hook_id}.txt"
+    if not path.is_file():
+        return set()
+
+    entries: set[tuple[str, str]] = set()
+    for raw_line in path.read_text(encoding="utf-8").splitlines():
+        line = raw_line.strip()
+        if not line or line.startswith("#"):
+            continue
+        match = _ENTRY_RE.match(line)
+        if match is None:
+            raise ValueError(f"Malformed baseline entry in {path}: {raw_line!r}")
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)